build-parent/pom.xml

```diff
@@ -86,7 +86,7 @@
 
         <!-- The image to use for tests that run Keycloak -->
         <!-- IMPORTANT: If this is changed you must also update .github/workflows/ci-actions.yml and bom/application/pom.xml to match the version -->
```

Comment:
You probably need to update this comment given you don't have to update `.github/workflows/ci-actions.yml` anymore.